src/components/BSelect/BSelect.test.js

```diff
@@ -148,18 +148,18 @@ describe('BSelect', () => {
         selectOptions: selectOptions,
         required: required,
         selectLabel: selectLabel,
-        placeholder: placeholder
-      }
-    })
-    const upHandler = jest.fn()
-    wrapper.vm.$on('upHandler', upHandler)
-    const div = wrapper.find(`#dropdown-${id}`)
-    div.trigger('mousedown')
-    await wrapper.vm.$nextTick()
-    const line = wrapper.find('li')
-    line.trigger('keyup.down')
-    expect(line.html()).toMatchSnapshot()
-    line.trigger('keyup.up')
-    expect(line.html()).toMatchSnapshot()
-  })
-})
+        placeholder: placeholder,
+      },
```

Comment:
`,` without something after it?

Reply:
Justification from AirBnB: https://github.com/airbnb/javascript#commas--dangling

My reasoning:
- easier to move things around
- simpler to parse git diffs
- `eslint --fix` captures it without you having to do anything
- standard practice in some other frameworks

Discuss

Reply:
I like trailing commas, but they do look weird.

Reply:
Agree, they threw me off at first, but I got used to them after a bit and miss them now